src/extension.ts

```diff
@@ -62,6 +62,9 @@ export function activate(context: vscode.ExtensionContext): void {
     ext.outputChannel = outputChannel;
     context.subscriptions.push(outputChannel);
 
+    // tslint:disable-next-line:prefer-const
+    let logsOutputChannels: { [channelName: string]: { channelObj: vscode.OutputChannel } } = {};
```

Comment:
If you're opposed to the `prefer-const` tslint rule I'd be curious to hear why, but otherwise please just use `const` instead of `let` rather than disabling the rule

Reply:
Changed to `const`. I was thinking that it's going to be reassigned but I see it works.